.gitignore

```diff
@@ -13,5 +13,6 @@ local
 cache
 *.cache
 *.cache.*
+*.cmd
```

Comment:
```suggestion
```

Reply:
no please, i need to exclude these to help me easily support multiple php versions at the same time in folder

Reply:
Use local .gitignore and revert this here.